test/suite/test_backup13.py

```diff
@@ -0,0 +1,368 @@
+            else:
+                copy_to = h
+
+            shutil.copy(copy_from, copy_to)
+            first = True
+            config = 'incremental=(file=' + newfile + ')'
+            dup_cnt = 0
+            incr_c = self.session.open_cursor(None, bkup_c, config)
+
+            # For each file listed, open a duplicate backup cursor and copy the blocks.
+            while True:
+                ret = incr_c.next()
+                if ret != 0:
+                    break
+                incrlist = incr_c.get_keys()
+                offset = incrlist[0]
+                size = incrlist[1]
+                curtype = incrlist[2]
+                # 1 is WT_BACKUP_FILE
+                # 2 is WT_BACKUP_RANGE
+                self.assertTrue(curtype == 1 or curtype == 2)
+                if curtype == 1:
+                    if first == True:
+                        h = self.home_incr + '.' + str(self.counter)
+                        first = False
+
+                    copy_from = newfile
+                    if ("WiredTigerLog" in newfile):
+                        copy_to = h + '/' + self.logpath
+                    else:
+                        copy_to = h
+                    shutil.copy(copy_from, copy_to)
+                else:
+                    self.pr('Range copy file ' + newfile + ' offset ' + str(offset) + ' len ' + str(size))
+                    write_from = newfile
+                    write_to = self.home_incr + '.' + str(self.counter) + '/' + newfile
+                    rfp = open(write_from, "r+b")
+                    wfp = open(write_to, "w+b")
+                    rfp.seek(offset, 0)
+                    wfp.seek(offset, 0)
+                    buf = rfp.read(size)
+                    wfp.write(buf)
+                    rfp.close()
+                    wfp.close()
+                dup_cnt += 1
+            self.assertEqual(ret, wiredtiger.WT_NOTFOUND)
+            incr_c.close()
+
+            # For each file, we want to copy the file into each of the later incremental directories
+            for i in range(self.counter, self.max_iteration):
+                h = self.home_incr + '.' + str(i)
+                copy_from = newfile
+                if ("WiredTigerLog" in newfile):
+                    copy_to = h + '/' + self.logpath
+                else:
+                    copy_to = h
+                shutil.copy(copy_from, copy_to)
+        self.assertEqual(ret, wiredtiger.WT_NOTFOUND)
+        bkup_c.close()
+
+    def compare_backups(self, t_uri):
+        #
+        # Run wt dump on full backup directory
+        #
+        full_backup_out = self.full_out + '.' + str(self.counter)
+        if self.counter == 0:
+            self.runWt(['-R', '-h', self.home, 'dump', t_uri], outfilename=full_backup_out)
+        else:
+            home_dir = self.home_full + '.' + str(self.counter)
+            self.runWt(['-R', '-h', home_dir, 'dump', t_uri], outfilename=full_backup_out)
```

Comment:
Similar to the above, could we just have the conditional set the variable and simplify this.

Reply:
Fixed